app/models/user.rb

```diff
@@ -4,14 +4,15 @@ class User < ActiveRecord::Base
   validates :first_name, :last_name, :display_name, :about_me, presence: true
   validates :display_name, format: { with: /\A[a-zA-Z]+\z/ }
   validates :display_name, :slug, uniqueness: true
+  validates :password, presence: true, on: :create
   validates :email,
             format: {
               with: /\b[A-Z0-9._%a-z\-]+@(?:[A-Z0-9a-z\-]+\.)+[A-Za-z]{2,4}\z/
             }, uniqueness: true
-  has_many :listings
   has_many :listing_images, :through => :listings
-  has_many :orders
-  validates :password, presence: true, on: :create
+  has_many :reservations
+  scope :host, -> (host) { where()}
```

Comment:
Unused block argument - `host`. If it's necessary, use `_` or `_host` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Do not use parentheses for method calls with no arguments.<br>Space missing inside }.